job_bundles/Using_Files_In_Jobs.md

```diff
@@ -0,0 +1,1161 @@
+a storage profile with the [CreateStorageProfile API](https://docs.aws.amazon.com/deadline-cloud/latest/APIReference/API_CreateStorageProfile.html),
+[AWS::Deadline::StorageProfile](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-deadline-storageprofile.html)
+[AWS CloudFormation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/Welcome.html) resource, or 
+[AWS Console](https://docs.aws.amazon.com/deadline-cloud/latest/userguide/storage-shared.html#storage-profile). 
+
+A Storage Profile is made up of a list of filesystem locations that each tell Deadline Cloud the location and type of a filesystem
+location that is relevant for jobs submitted from or run on a host. A storage profile should only model the locations that are
+relevant for jobs. For example, the shared `FSComm` location is located on workstation `WS1` at `S:\`, so the corresponding
+filesystem location is:
+
+```json
+{
+    "name": "FSComm",
+    "path": "S:\\",
+    "type": "SHARED"
+}
+```
+
+Now, create the storage profile for workstation configurations `WS1`, `WS2`, and `WS3` and the worker configuration `WorkerCfg`
+using the [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-welcome.html) in
+[AWS CloudShell](https://docs.aws.amazon.com/cloudshell/latest/userguide/welcome.html):
+
+```bash
+# Change the value of FARM_ID to your farm's identifier
+FARM_ID=farm-00112233445566778899aabbccddeeff
+
+aws deadline create-storage-profile --farm-id $FARM_ID \
+  --display-name WSAll \
+  --os-family LINUX \
+  --file-system-locations \
+  '[
+      {"name": "FSComm", "type":"SHARED", "path":"/shared/common"},
+      {"name": "FS1", "type":"SHARED", "path":"/shared/projects/project1"},
+      {"name": "FS2", "type":"SHARED", "path":"/shared/projects/project2"}
+  ]'
+
+aws deadline create-storage-profile --farm-id $FARM_ID \
+  --display-name WS1 \
+  --os-family WINDOWS \
+  --file-system-locations \
+  '[
+      {"name": "FSComm", "type":"SHARED", "path":"S:\\"},
+      {"name": "FS1", "type":"SHARED", "path":"Z:\\"}
+   ]'
+
+aws deadline create-storage-profile --farm-id $FARM_ID \
+  --display-name WS2 \
+  --os-family MACOS \
+  --file-system-locations \
+  '[
+      {"name": "FSComm", "type":"SHARED", "path":"/Volumes/common"},
+      {"name": "FS2", "type":"SHARED", "path":"/Volumes/projects/project2"}
+  ]'
+
+aws deadline create-storage-profile --farm-id $FARM_ID \
+  --display-name WorkerCfg \
+  --os-family LINUX \
+  --file-system-locations \
+  '[
+      {"name": "FSComm", "type":"SHARED", "path":"/mnt/common"},
+      {"name": "FS1", "type":"SHARED", "path":"/mnt/projects/project1"},
+      {"name": "FS2", "type":"SHARED", "path":"/mnt/projects/project2"}
+  ]'
+```
+
+It is essential that the file system locations in your storage profiles are referenced using the same values
```

Comment:
Nit: any way we can add something eye catching so we don't miss these kinds of important tips? Maybe just an emoji (unless that's tacky) ⚠️

Reply:
Modified.